packages/headless/doc-parser/src/interface-resolver.ts

```diff
@@ -62,12 +62,36 @@ function filterOverridesAndCombine(
   return members.concat(filtered);
 }
 
+function sortEntities(entities: AnyEntity[]) {
+  const kindOrder: Record<EntityKind, number> = {
+    primitive: 0,
+    'primitive-with-type-alias': 0,
+    object: 0,
+    function: 1,
+  };
+
+  return entities.sort((a, b) => {
```

Comment:
I chose the order:
* Mandatory attributes
* Optional attributes
* Methods

Do you have any other preference?